pkg/injector/webhook.go

```diff
@@ -332,6 +363,25 @@ func isAnnotatedForInjection(annotations map[string]string, objectKind string, o
 	return
 }
 
+func isAnnotatedForOutboundPortExclusion(annotations map[string]string, objectKind string, objectName string) (exists bool, ports []string, err error) {
+	outboundPortsToExclude := strings.ToLower(annotations[constants.OutboundPortExclusionListAnnotation])
```

Comment:
We should change the logic as follows:
1. check if the annotations map has the key
2. if key is missing, return an empty list of ports
3. Parse the comma separated list of ports and return

This will avoid the additional identation here.